### DISCUSSION TITLE
Tray icons show/hide not working anymore

### DISCUSSION THREAD

**Discussion by xxrail** [valinet/ExplorerPatcher#1172] [General]
Hello,
with the latest win11 update the option that allow to hide or show specific tray icon wont work anymore.
All icons are off expect system icons.

Edition	Windows 11 Enterprise
Version	22H2
Installed on	‎20/‎03/‎2023
OS build	22621.1413
Experience	Windows Feature Experience Pack 1000.22639.1000.0

![image](https://user-images.githubusercontent.com/85962636/226846973-ac58ae09-de0a-4446-9dcd-342ee2c7b96d.png)

**Comment by Jonny-T-cyber** [comment]
Hi!
I have the same problem...
No solution?

**Reply by topia** [reply]
I think you can drag and drop icons.

**Reply by Eowyn666** [reply]
Thanks a ton! It was driving me nuts!

**Comment by jdh23662** [comment]
This appears to be a valid issue with EFP build 22621.1413.  If I uninstall EFP or roll back to 22621.1265, every thing works as it should.  Also, by using the link in the properties page, the old icon tray options opens but no changes are applied.

![image](https://user-images.githubusercontent.com/69373422/227810024-5268239a-85b9-41f2-bbc3-2287a2fd4304.png)

So, do the devs look here or is there an official vehicle for reporting this?

**Reply by topia** [reply]
Devs check discussions. By the way, the last windows update causes many issues and there is not enough time to find all solutions. and we have a workaround for this.

**Reply by valinet** [reply]
The backend for system tray icons changed - the settings are now stored in a different place in the registry. Of course, legacy stuff, like the old taskbar, old Control Panel applet etc, only know about and use the former place. If you use the Windows 11 Settings app to reorder icons, this writes changes in the new data store, which the legacy stuff has no idea about...
